using_listwheelscrollview/lib/main.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'listwheel.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData.dark().copyWith(
+        scaffoldBackgroundColor: Color(0XFF0A0E21),
+      ),

```

Comment:
Removing theme keeps the app same. So remove this.

Reply:
ok sir